src/main/java/com/gdschongik/gdsc/domain/study/domain/StudyValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.gdschongik.gdsc.domain.study.domain;
+
+import static com.gdschongik.gdsc.global.exception.ErrorCode.STUDY_MENTOR_INVALID;
+import static com.gdschongik.gdsc.global.exception.ErrorCode.STUDY_MENTOR_IS_UNAUTHORIZED;
+
+import com.gdschongik.gdsc.domain.member.domain.Member;
+import com.gdschongik.gdsc.global.annotation.DomainService;
+import com.gdschongik.gdsc.global.exception.CustomException;
+
+@DomainService
+public class StudyValidator {
+    public void validateStudyMentor(Member currentMember, Study study) {
+        // 어드민인 경우 검증 통과
+        if (currentMember.isAdmin()) {
+            return;
+        }
+
+        // 어드민이 아니고 멘토 역할도 아니면 경우에 예외가 밸생합니다.
+        if (!currentMember.isAdmin() && !currentMember.isMentor()) {
```

Comment:
```suggestion
        if (!currentMember.isMentor()) {
```
위의 조건에서 어드민이 아니다(!currentMember.isAdmin())는 반드시 참입니다